package.json

```diff
@@ -58,7 +59,7 @@
   },
   "dependencies": {
     "@types/react": "^16.x",
-    "monaco-editor": "*",
+    "monaco-editor": "0.19.3",
```

Comment:
This makes the module incompatible with 0.20.

Reply:
whoops, this was left in there from experimenting I was doing, will change back